pyvrp/GeneticAlgorithm.py

```diff
@@ -32,33 +32,33 @@ class GeneticAlgorithmParams:
         Probability (in :math:`[0, 1]`) of repairing an infeasible solution.
         If the reparation makes the solution feasible, it is also added to
         the population in the same iteration.
-    nb_iter_no_improvement
+    num_iter_no_improvement
```

Comment:
I have a preference for `num_iters_no_improvement`, similar to how we call it `iters_no_improvement` in GA.